packages/p2p/src/components/modal-manager/modals/loading-modal/__tests__/loading-modal.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ModalManagerContextProvider from 'Components/modal-manager/modal-manager-context-provider';
+import LoadingModal from '../loading-modal';
+
+jest.mock('Components/modal-manager/modal-manager-context', () => ({
+    useModalManagerContext: () => ({
+        is_modal_open: true,
+    }),
+}));
+
+describe('<LoadingModal />', () => {
+    let modal_root_el: HTMLElement;
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+    it('should render loading modal', () => {
+        render(<LoadingModal />);
+        expect(screen.getByTestId('dt_initial_loader')).toBeInTheDocument();
```

Comment:
hi, if you check this [document](https://testing-library.com/docs/queries/about/#priority) you will see that getByTestId has the lowest priority in the all the selectors. I recommend you to use getByRole or getByLabelText and ... if it's possible.

Reply:
Thanks @ali-hosseini-deriv , I will check if i can use other selector 🙏

Reply:
there would be an icon or something similar to that in the DOM right?
maybe you can use that.

Reply:
it only contains divs without any text 😢 

![image](https://github.com/farrah-deriv/deriv-app/assets/125247833/38a58955-da11-4644-b031-288db827dd11)